frontend/lib/src/components/widgets/DataFrame/DataFrame.tsx

```diff
@@ -126,6 +126,8 @@ export interface DataFrameProps {
   disableFullscreenMode?: boolean
   fragmentId?: string
   height?: number
```

Comment:
Is the height here still relevant for something? E.g. is this passed in for fullscreen mode?